src/frontend.md

```diff
@@ -221,6 +221,24 @@ export function fakeTicket() {}
 
 1. If some color is repeated several times in code, extract it to theme.
 
+## Disposition of components
+
+- Prefer using local components `(host|mf)/src/components` with a few exceptions
+- Use UI Kit distribution if a component represents a design token (1) [websome](<https://www.figma.com/file/I8WglqziPB5pYlFIqF43JE/Client-UI-Kit-(Web)?node-id=0-1&t=MuFhhOjUXP4sFRr2-0>) or matches with [MUI](https://mui.com/material-ui/)
+- Use MF distribution if a component is required in several repositories and could be shipped as a whole piece with API
```

Comment:
We have a core functionality and sometimes we want to reuse it. in this case, MF is not a good choice. Websome kit is an example of it.
My experience talks me that we should isolate and group all domains in MFs excluding core, "onboarding" or/and sales.

Reply:
Updated